nxlog.sls

```diff
@@ -1,10 +1,28 @@
 nxlog:
-  '2.10.2150':
+  '2.10.2100':
```

Comment:
2150

Reply:
Sorry about that, I just checked in the change